__tests__/helpers/requestMock.ts

```diff
@@ -0,0 +1,30 @@
+import { ClientRequest, request } from "http";
+import IntegrationTestEnvironment from "./integrationTestEnvironment";
+
+export default function requestMock(
+  env: IntegrationTestEnvironment,
+  mockID: string,
+  method: "GET" | "POST" = "GET",
+  body?: any,
+  contentType: string = "application/json"
+): ClientRequest {
+  const req = request({
+    port: env.port,
+    hostname: "localhost",
+    method,
+    headers: {
+      Cookie: `MockserverID=${mockID}`,
+      "Content-Type": contentType
+    }
+  });
+
+  if (method === "POST") {
+    const buffer = Buffer.from(body);
```

Comment:
`body` is optional but we don't check it. Plus `Buffer.from` requires `first argument must be a string, Buffer, ArrayBuffer, Array, or array-like object.`

Reply:
Good catch, I'll add the resp. types as well as a null check.

Reply:
✔️  Done